js/models/BaseModel.js

```diff
@@ -213,7 +227,11 @@ export default class extends Model {
   }
 
   clone() {
-    const clone = new this.constructor(this.toJSON(), { guid: this.guid });
+    let clone;
```

Comment:
@rmisio this key deletion thing was happening when I started to use nested models. `toModelFormatJSON` is part of the monkey patch to get around it. Without the monkey patching, the advanced settings model gets serialized to `{}`. Either the nested models code was incomplete or I do not know how to use it correctly. More likely the latter, tho my feeling is it's the former. I see that it would have worked for me to trigger the PR earlier in the process -- rather than trying to monkey patch my way through. @jjeffryes and @rmisio are being very helpful, I really appreciate that and wasn't necessarily counting on it. So next time, I think, I'll trigger a PR earlier if I meet something that isn't so clear.